grunt_tasks/webpack.config.js

```diff
@@ -69,7 +70,14 @@ module.exports = {
             'window.jQuery': 'jquery'
         }),
         // Disable writing the output file if a build error occurs
-        new webpack.NoEmitOnErrorsPlugin()
+        new webpack.NoEmitOnErrorsPlugin(),
+        new GoogleFontsPlugin({
```

Comment:
Much like fontello, there is a passing reference to it in `clients/web/static/built/girder_lib.min.css`, but the main css file gets loaded by itself (neither is bundled), which is why it is explicitly referenced in the webroot.mako file.

I do see that the GoogleFontsPlugin is creating a css file for the main and per plugin, which is wasteful.